netbout-web/src/main/java/com/netbout/dynamo/DyBase.java

```diff
@@ -50,6 +50,7 @@
 @Loggable(Loggable.DEBUG)
 @ToString(of = "region")
 @EqualsAndHashCode(of = "region")
+@SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
```

Comment:
@mkordas No it is not 0.16.2. It is related to 0.15.2, so I will try to fix it and remove the SuppressWarnings annotation.